_topic_map.yml

```diff
@@ -1579,6 +1579,9 @@ Topics:
   Topics:
   - Name: Getting started with Knative CLI
     File: knative-cli
+### HA    
+- Name: High availability (HA) using OpenShift Serverless
```

Comment:
I think we might want to adjust wording here, this seems to imply HA of your services, when in fact it is the control plane which is HA

Reply:
I tried to find a similar topic in the OpenShift docs and couldn't, curiously